lighthouse-core/gather/computed/critical-request-chains.js

```diff
@@ -38,6 +38,10 @@ class CriticalRequestChains extends ComputedArtifact {
     if (resourceTypeCategory === WebInspector.resourceTypes.XHR._category) {
       return false;
     }
+    // Images are also non-critical.
+    if (resourceTypeCategory === WebInspector.resourceTypes.Image._category) {
```

Comment:
@abacon Sorry for stealing your thunder on the follow-up PR. I covered it in #2191 already. Thanks for the PR 👍